app/assets/javascripts/current_user.coffee

```diff
@@ -0,0 +1,10 @@
+class User
+  constructor: (@data = {}) ->
+
+  get: (param) ->
+    @data[param]
+
+  isSignedIn: ->
+    @get("id")?
+
+@currentUser = App.currentUser = new User(App.currentUserData)
```

Comment:
I personally don't like having global variables in JS: it's easy to spoil it by forgetting to put `var` before. But it's definitely much shorter than `App.currentUser` and I saw a lot of people using it like that.

If you also don't like global variables let's leave only `App.currentUser`. Developers can make it global in any time.